package.json

```diff
@@ -22,14 +22,15 @@
     "format": "npm run prettier -- --write",
     "format:ci": "npm run prettier -- --check",
     "integration": "npm run build && npm run nobuild:integration",
+    "lint": "tslint -p tsconfig.json -c tslint.json",
     "nobuild:coverage-report": "nyc report --reporter=lcov",
     "nobuild:coverage-html": "nyc report --reporter=html",
     "nobuild:coverage": "nyc --check-coverage --branches 90 --functions 90 --lines 90 mocha",
     "nobuild:integration": "mocha --timeout 50000 test/integration/",
     "nobuild:unit": "mocha",
     "prepare": "npm run build",
     "prettier": "prettier ts_src/*.ts ts_src/**/*.ts --ignore-path ./.prettierignore",
```

Comment:
add `ts_src/**/**/*.ts` to this so that the templates also are formatted.

Reply:
fixed
https://github.com/bitcoinjs/bitcoinjs-lib/pull/1353